insights/parsers/ls_boot.py

```diff
@@ -38,20 +38,19 @@
     True
 """
 
-from .. import FileListing, parser, CommandParser
+from insights import CommandParser, FileListing, parser
```

Comment:
@xiangce Can you use the new FileListing [function](https://github.com/RedHatInsights/insights-core/blob/master/insights/parsers/ls.py#L44) that was added for these parsers? If not the deprecation would need to also be removed from the old FileListing [function](https://github.com/RedHatInsights/insights-core/blob/master/insights/core/__init__.py#L1644).

Reply:
@ryan-blakley - Yeap, I missed this. Fixed in the new commit.

Thanks for reviewing.